### ISSUE TITLE
The while command in a shell script doesn't stop by Ctrl+C.

### ISSUE THREAD

**Issue by ryuichiueda** [shellgei/rusty_bash#110]
This script never stops by Ctrl+C while it stops by `killall hoge.sush`. 

```bash
🍣 cat hoge.sush
#!/bin/sush -xv

seq 100 |
while read a ; do
	sleep 1
	echo $a
done
```

**Comment by ryuichiueda** [comment]
The reason of this bug is that the current implementation reads a shell script from stdin. In this case, the connection between tty and stdin is disconnected. Therefore, the foreground is not set to the while command.

**Comment by ryuichiueda** [comment]
Solved on v0.9.0
